packages/components/stamp/README.md

```diff
@@ -44,9 +44,11 @@ export default Example;
-| `tone`              | `union`<br/>Possible values:<br/>`, 'critical', 'warning', 'positive', 'information', 'primary', 'secondary'` |          |         | Indicates the color scheme of stamp                                                                                    |
-| `isCondensed`       | `boolean`                                                                                                     |          | `false` | If `true`, renders a condensed version of the stamp.                                                                   |
-| `children`          | `ReactNode`                                                                                                   |    ✅    |         |                                                                                                                        |
-| `overrideTextColor` | `boolean`                                                                                                     |          | `false` | If true, the css text color for any child of the Stamp component will be overriden by the one set from the Stamp prop. |
+| Props         | Type                                                                                                          | Required | Default         | Description                                          |
+| ------------- | ------------------------------------------------------------------------------------------------------------- | :------: | --------------- | ---------------------------------------------------- |
+| `tone`        | `union`<br/>Possible values:<br/>`, 'critical', 'warning', 'positive', 'information', 'primary', 'secondary'` |          | `'information'` | Indicates the color scheme of stamp                  |
+| `isCondensed` | `boolean`                                                                                                     |          | `false`         | If `true`, renders a condensed version of the stamp. |
+| `children`    | `ReactNode`                                                                                                   |          |                 |                                                      |
+| `icon`        | `ReactElement`                                                                                                |          |                 |                                                      |
+| `label`       | `string`                                                                                                      |          |                 | used as an alternative to `intlMessage` prop         |
+| `intlMessage` | `intersection`                                                                                                |          |                 |                                                      |
```

Comment:
we need to re-generate the readme ♻️

Reply:
Thanks! I missed that part. 
Updated here: https://github.com/commercetools/ui-kit/pull/2383/commits/0a0ead4b3f8d3b5ff707c972d064b93470513995